spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/bedrock/jurrasic2/BedrockAi21Jurassic2ChatAutoConfiguration.java

```diff
@@ -37,13 +37,14 @@
  *
  * @author Ahmed Yousri
  * @author Wei Jiang
+ * @author Soby Chacko
  * @since 1.0.0
  */
 @AutoConfiguration
 @ConditionalOnClass(Ai21Jurassic2ChatBedrockApi.class)
 @EnableConfigurationProperties({ BedrockAi21Jurassic2ChatProperties.class, BedrockAwsConnectionProperties.class })
 @ConditionalOnProperty(prefix = BedrockAi21Jurassic2ChatProperties.CONFIG_PREFIX, name = "enabled",
-		havingValue = "true")
+		havingValue = "true", matchIfMissing = true)
```

Comment:
Because, currently,  we are using a single boot starter for all Bedrock providers, the Bedrock models should be [disabled by default](https://docs.spring.io/spring-ai/reference/api/bedrock-chat.html#_enable_selected_bedrock_model).